src/components/FormValidator.js

```diff
@@ -77,4 +85,12 @@ export default class FormValidator {
     this._setEventListeners();
     this._toggleButtonState();
   }
+
+  renderLoading(isLoading) {
```

Comment:
`renderLoading` should be in `ModalWithForm` because `FormValidator` is only for validation